requirements.txt

```diff
@@ -1,20 +1,22 @@
 altair
-hdbscan==0.8.33
-metaflow==2.9.14
-numpy==1.23.4
+hdbscan
```

Comment:
Yes... I have already forgotten exactly what errors but yes 🙈 I removed all the package versions (except for the bigquery one, because I didn't want to tamper with that) and initialised the environment with python 3.10